apis/models/storedPinDao.js

```diff
@@ -0,0 +1,60 @@
+const database = require('./dataSource');
+const queryRunner = database.createQueryRunner();
+
+const createStoredPin = async (userId, pinId) => {
+  try {
+    const result = await database.query(
+      `
+    INSERT INTO stored_pins (
+        user_id, pin_id
+    ) VALUES (
+         ?, ?
+    )
+    `,
+      [userId, pinId]
+    );
+    return result;
+  } catch (error) {
+    if (error.code.includes('DUP_ENTRY')) {
+      error.statusCode = 400;
+      error.message = 'ALREADY_STORED_PIN';
+      throw error;
+    }
+    if (error) {
+      error.statusCode = 400;
+      error.message = 'WRONG_INPUT_DATA';
+      throw error;
+    }
```

Comment:
case 별로 에러 메세지 혹은 status code를 달리하여 핸들링하시는 걸 의도하셨다면 catch 블럭 마지막에서 throw는 한번만 해주셔도 될 거 같네요!
```suggestion
  } catch (error) {
    if (error.code.includes('DUP_ENTRY')) {
      error.statusCode = 400;
      error.message = 'ALREADY_STORED_PIN'
    }
    if (error) {
      error.statusCode = 400;
      error.message = 'WRONG_INPUT_DATA';
    }
    throw error;
```

Reply:
수정하였습니다.